css/build.css

```diff
@@ -115,4 +115,77 @@
 
 .close-icon {
   width: 40%;
+}
+
+/*
+  This part of the code is needed for supporting .fa-times-thin class in users who updated
+  to use UNICODE icons instead of SVG icons.
+*/
+
+.fa-times-thin {
```

Comment:
I think it's best to use a selector alongside this, e.g. `.close-icon .fa-times-thin`. We don't want all of `.fa-times-thin` anywhere in the app to be affected -- only if it's used in the LFD within the expected detail view overlay.

Reply:
@tonytlwu 
Added more specified CSS selector